exception=pmc_funccall_internal
